src/Interface/Controllers/movie.controller.ts

```diff
@@ -57,10 +57,17 @@ export class MovieController {
     return this.movieService.getMovies();
   }
 
-  @MessagePattern('get_all_movies')
+  @Get('es')
+  @ApiOperation({ summary: 'Send all movies to ES' })
+  @ApiResponse({
+    status: 200,
+    description: 'Gets all movies from database and sends them to ES via RabbitMQ',
+    type: '',
+    isArray: true,
+  })
   async getMoviesEs() {
-    const response = this.movieService.getMovies();
-    this.client.emit('movies_list', response);
+    const response = await this.movieService.getMovies();
+    this.client.emit('InsertMovies', response);
```

Comment:
- InsertMovies --> movies_list or insert_movies. naming convention rabbitMQ: https://www.rabbitmq.com/tutorials/tutorial-five-go

Reply:
Nestjs naming convention, but we will use insert_movies